Chatto/Source/ChatController/BaseChatViewController+Changes.swift

```diff
@@ -71,13 +71,14 @@ extension BaseChatViewController {
             return 0
         }
 
-        let contentHeight = self.collectionView.contentSize.height
```

Comment:
The most of functions that access `collectionView` don't make sense to call if it wasn't created yet. Could it be better if you use `guard` statement at the beginning to get unwrapped `collectionView`?

Reply:
Done